.CI/travis_fast_finish.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
Still thinking this should be in `conda-forge-build-setup`. However given this script no longer needs anything beyond a stock Python interpreter, we may just `curl` the script instead of trying to use `conda` to install it. This way we can still run it as early as possible.